packages/openactive-integration-tests/documentation/generator.js

```diff
@@ -15,7 +15,7 @@ const { FeatureJsonSchema } = require('./featureJson.js');
 
 const FEATURES_ROOT = path.join(__dirname, '..', 'test', 'features');
 const INDEX_README_FILE = path.join(FEATURES_ROOT, 'README.md');
-const INDEX_CRITERIA_REQUIREMENTS_JSON_FILE = path.join(FEATURES_ROOT, 'criteria-requirements.json');
+const INDEX_FEATURE_REQUIREMENTS_JSON_FILE = path.join(FEATURES_ROOT, 'feature-requirements.json');
```

Comment:
Renamed because it now contains more info. But this new info (required TI Actions) is still per-feature. So, feature-requirements

Reply:
If the reviewer is not aware, Test Data Generator gets its metadata from this file which is generated during automatic documentation generation (which has access to each test's describeFeature(..) metadata)